### PR TITLE
Marine Quirk + Crayon eating nerf (Happy?)

### PR DESCRIPTION
**Pull request by Therandomhoboo**
# Document the changes in your pull request
Thanks Drake. Joking aside, it was pointed out that crayons give nutriments of 5-6u's and rainbows give 15-20u's. Which is a very good food staple that is a tiny item which can be easily stacked/stored. We shouldn't be fostering this meta and so to combat this, I've given crayons a disgust stack. Taking bites out of them gives you disgust so if you binge on them. You end up vomitting.
[EDIT] 
I've now added in a new quirk since so many people got hurt, it is called the Marine Quirk. It gives you the amazing ability to not get disgust. Nice! It costs 2 because you made me do this.

@dcxgame for making the Marine icon for this but cause I couldn't get it in (my fault) they found a chevron badge in the code I can use instead.

# Why is this good for the game?

Stops ~~drake~~ players from bypassing eating actual food by giving disgust per bite. Crayons are nasty anyway.
[EDIT]
Gives players the fucking Quirk so they will stop downvoting

# Wiki Documentation

Need Marine Quirk added to the section. Name: Marine, Balance: 2, Gameplay effects: You can eat crayons without puking hooray. Mood locked: X

# Testing

Eating a full crayon usually makes the player puke, but it applies disgust per bite they take properly. 
[EDIT]
All works

# Changelog

:cl:  

rscadd: The stupid Marine Quirk to let you eat fucking crayons
rscadd: Made taking bites of crayon give you stacks of disgust because you're eating wax and shit.

/:cl:

### CODE REVIEW

**Comment by LaOubliette** [comment]
I will weaponize this, fool. You cant stop chemistry.

**Comment by 00ze-cyclone** [comment]
Can’t you just grind it anyway? This solve pretty much nothing if so.

**Comment by cowbot92** [comment]
this dude has never eaten a delicious crayon irl

**Comment by Therandomhoboo** [comment]
> I will weaponize this, fool. You cant stop chemistry.


> Can’t you just grind it anyway? This solve pretty much nothing if so.

Then they can carry it in a beaker and chug it instead of having 12 packs of 6 crayons. This just stops them stacking it more so with no effort, if they want to take time and grind them all up crack on.

[EDIT] Also want to point out that as well, you're not always near a grinder or got the stuff to make one. So it means if people want to grind it instead then they have to do it beforehand or have the right materials to make a grinder. So yeah, its a win-win.

**Comment by LassiH** [comment]
> [EDIT] Also want to point out that as well, you're not always near a grinder or got the stuff to make one. So it means if people want to grind it instead then they have to do it beforehand or have the right materials to make a grinder. So yeah, its a win-win.

said mortar and pestle in players bag

**Comment by Therandomhoboo** [comment]
> > [EDIT] Also want to point out that as well, you're not always near a grinder or got the stuff to make one. So it means if people want to grind it instead then they have to do it beforehand or have the right materials to make a grinder. So yeah, its a win-win.
> 
> said mortar and pestle in players bag

....fuck

**Comment by RG4ORDR** [comment]
Im a Marine, you cannot do this.
Cultural erasure.

**Comment by Therandomhoboo** [comment]
YOU CAN STILL EAT THE GOD DAMN CRAYON GUYS, I WANT TO MAKE THIS **ABUNDANTLY** CLEAR

**Comment by UltraBigRat** [comment]
Remove the disgust for sec, crayoncurity meta.

**Review by JohnFulpWillard — COMMENTED** [review]
this is discouraging crayon-eaters from eating their god-given food.
security should be exempt

**Comment by Therandomhoboo** [comment]
> this is discouraging crayon-eaters from eating their god-given food. security should be exempt

If balance council says sec should be exempt i'll exempt those mf-ers

**Comment by ThatGuyPhil** [comment]
Maybe give them less nutriment? You can get fat from eating less than half a pack of crayons. I think it would be a better nerf than making them taste bad.

**Comment by RG4ORDR** [comment]
Mime,Clown,Sec, and Captain should all get benefits from eating crayons.
simple as.

**Review by ynot01 — APPROVED** [review]

**Comment by Therandomhoboo** [comment]
I am putting this on hold, for the time being until a quirk is done for the fucking marines.

**Comment by Therandomhoboo** [comment]
There you fucking go, its in the works.

**Comment by Therandomhoboo** [comment]
It is all finished, it all works.

File: code/__DEFINES/traits.dm [hunk 1]
```diff
@@ -272,6 +272,8 @@
 #define TRAIT_BADMAIL			"badmail"	//Your mail is going to be worse than average
 #define TRAIT_SHORT_TELOMERES	"short_telomeres" //You cannot be CLOONED
 #define TRAIT_LONG_TELOMERES	"long_telomeres" //You get CLOONED faster!!!
+///You become Marine!!!
```
Comment (Moltijoe) [inline code/__DEFINES/traits.dm]: please say what the trait actually does

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/__DEFINES/traits.dm (hunk 1 above)
Comment (Therandomhoboo) [reply code/__DEFINES/traits.dm]: Amended

**Review by Moltijoe — APPROVED** [review]
good enough

**Comment by Yarinoi** [comment]
yes we are

**Comment by Runian** [comment]
Crayon disgust 👍 Marine quirk 👎

**Comment by Therandomhoboo** [comment]
> Crayon disgust 👍 Marine quirk 👎

I had to do this to sate the Marines that were encroaching quickly on my location

**Comment by RG4ORDR** [comment]
Merge this on the USMC Birthday please.

**Pull request merged**